lib/templates/error.html

```diff
@@ -1085,14 +1085,16 @@
     </header>
     <main class="container">
       <article class="content">
+        <div onclick="fetch('/ember_open_editor_for_template_compilation_error')">
```

Comment:
what about browsers that don't support `fetch()`?

Reply:
Good catch 👍